extensions/swagger-ui/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-swagger-ui-parent</artifactId>
+    <name>Quarkus - Swagger UI</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <swagger-ui.version>3.20.9</swagger-ui.version>
```

Comment:
This needs to be added to the BOM (together with the dependency declaration).

Reply:
I've an issue between the `bom` and `build-parent` management.
The `bom` defines the dependencies but I can inherit a property. I can do it with the `build-parent.
I need this property for a path construction
https://github.com/quarkusio/quarkus/pull/1573/files#diff-021ff459d881cd8da34e1d20241adb8cR77

ATM, I've the `swagger-ui.version` version in both poms... Do you see any proper way to do that?

Reply:
Hmmm, I missed that. I checked both plugins for which you used the version and I think you could make it work with an `*` as some of the directives of both plugins support that.

At least I think it's worth a try.

Reply:
:thinking:  not sure to understand your comment? You mean define the dependency version in the `bom` and remove the property from the `build-parent` + replace the replace `${swaggerui.version}` by `*` like `${project.build.directory}/webjar-swagger-ui/META-INF/resources/webjars/swagger-ui/*/` ?

Reply:
Yes, exactly. I think it could work as at least some of the properties support `*`, for both plugins.

Reply:
With @stuartwdouglas proposal, we don't need this anymore.